app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/ResourcesGridPresenter.kt

```diff
@@ -52,7 +52,7 @@ class ResourcesGridPresenter(
         val resource = selection[view.position()]
 
         val path = index.getPath(resource)
-            ?: throw AssertionError("Resource to display must be indexed")
+            ?: return
```

Comment:
`AssertionError`s should be removed only when their invariants don't hold anymore.